src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --color1: #00acdc;
+  --color2: #000;
+  --blockheight: 4px;
+  --blocktopgap: 26.5px;
+  --blockwidth: 37px;
+  --border1: 8px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 5px 0;
+  color: var(--color2);
+}
+
+.apple,
+.laptops {
+  color: var(--color1);
+}
+
+.list {
```

Comment:
Use BEM - `nav__list`

Btw, your `ul` has some padding on the left side. Try to get rid of that.